pangeo-notebook/conda-lock.yml

```diff
@@ -30495,11 +30893,11 @@ package:
     tqdm: ''
     trajan: ''
     xarray: '>=0.10.8'
-    zarr: '>=2.11.0,!=2.18.0'
-  url: https://conda.anaconda.org/conda-forge/noarch/parcels-3.1.1-pyhea20964_1.conda
+    zarr: '>=2.11.0,!=2.18.0,<3'
```

Comment:
_Originally posted by @weiji14 in https://github.com/pangeo-data/pangeo-docker-images/pull/595#discussion_r1915501711_

> `parcels` has pinned to `zarr<3` at https://github.com/conda-forge/parcels-feedstock/pull/129, need to keep an eye on https://github.com/OceanParcels/Parcels/issues/1819

Do we want to downgrade from zarr 3.0.0 to 2.18.3?